core/libs/shared/src/core_ops.rs

```diff
@@ -146,22 +147,6 @@ where
         Ok((Some(file), id))
     }
 
-    pub fn create_unvalidated(
-        &mut self, parent: &Uuid, name: &str, file_type: FileType, account: &Account,
-    ) -> SharedResult<Uuid> {
-        let (op, id) = self.create_op(parent, name, file_type, account)?;
-        self.stage_and_promote(op);
-        Ok(id)
-    }
-
-    pub fn create(
-        &mut self, parent: &Uuid, name: &str, file_type: FileType, account: &Account,
-    ) -> SharedResult<Uuid> {
-        let (op, id) = self.create_op(parent, name, file_type, account)?;
-        self.stage_validate_and_promote(op, Owner(account.public_key()))?;
-        Ok(id)
-    }
-
```

Comment:
these and similar functions moved to an `impl` block which requires `Local` to be `TreeLikeMut` (this impl block modified to accept a `Local` that's only `TreeLike`)